lib/firefox/webdriver/index.js

```diff
@@ -9,6 +9,7 @@ const isEmpty = require('lodash.isempty');
 const get = require('lodash.get');
 const defaultFirefoxPreferences = require('./firefoxPreferences');
 const disableSafeBrowsingPreferences = require('./disableSafeBrowsingPreferences');
```

Comment:
Ok great, I didn't realize that we already had an option to disable SafeBrowsing.